app/scheduler/modules/pairing.py

```diff
@@ -0,0 +1,111 @@
+"""Match two Launch Pad member for a private conversation"""
+from slack import WebClient
+from interface.slack import Bot
+from random import shuffle
+from .base import ModuleBase
+from typing import Dict, List, Any, Set
+from flask import Flask
+from config import Config
+from db.facade import DBFacade
+from app.model import Pairing
+import logging
+
+
+class PairingSchedule(ModuleBase):
+    """Module that matches 2 Launch Pad members each week"""
+
+    NAME = 'Match launch pad members randomly'
```

Comment:
```suggestion
    NAME = 'Pair members randomly for meetups'
```

(pipe dream for https://github.com/ubclaunchpad/rocket2/issues/203 🙃 )